src/icons/IconMuseum.tsx

```diff
@@ -17,3 +17,5 @@ export default ({ className = '' }: Props): ReactElement => (
     </g>
   </svg>
 );
+
+export default IconMuseum;
```

Comment:
export default change is fine for me, but I don't understand what's the difference between these 2.

Reply:
Eslint started comlaining about these after updates and named default exports are better so they are not anonymous functions.

Reply:
For example Nextjs complains about this by default

Reply:
with `export default () => {}` function won't have any name in callstack when for example error happens and callstack is logged.

Reply:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-anonymous-default-export.md